scripts/variantstore/wdl/extract/hail_gvs_import.py

```diff
@@ -41,6 +41,12 @@ def import_gvs(argsfn, vds_path, references_path, temp_path):
         reference_genome=rg38,
         final_path=vds_path,
         tmp_dir=f'{temp_path}/hail_tmp_import_gvs',
+        # truth_sensitivity_snp_threshold: 'float' = 0.997,
+        # truth_sensitivity_indel_threshold: 'float' = 0.990,
+        # partitions_per_sample=0.35,
+        # intermediate_resume_point=0,
+        # skip_final_merge=false,
+        # unphase=false,
```

Comment:
Why is this here?

Reply:
so that if you want to adjust this python script a bit for different defaults (eg you dont want phasing info) you can see this comment instead of looking in Tim's source code